package-lock.json

```diff
@@ -1,29 +1,28 @@
 {
     "name": "@oat-sa/tao-item-runner-qti",
-    "version": "0.6.0",
+    "version": "0.6.1",
     "lockfileVersion": 1,
     "requires": true,
     "dependencies": {
         "@babel/code-frame": {
-            "version": "7.5.5",
-            "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.5.5.tgz",
-            "integrity": "sha512-27d4lZoomVyo51VegxI20xZPuSHusqbQag/ztrBC7wegWoQ1nLREPVSKSW8byhTlzTKyNE4ifaTA6lCp7JjpFw==",
+            "version": "7.8.3",
```

Comment:
Caution, despite dependencies have not changed in the `package.json` file, you have a lot of changes in the lock file.